### PR TITLE
fix(search) - tag value performance

### PR DESCRIPTION
**Pull request by wmak**
- Currently when a user enters a tag into a search, as they type the tags value we try to auto complete the tag value for them. As they type the API call currently doesn't include a timeframe, which results in us querying 90 days by default.
- This adds a timeframe to these queries which should help improve the performance of our queries.

### CODE REVIEW

File: src/sentry/static/sentry/app/components/smartSearchBar/index.jsx [hunk 1]
```diff
@@ -482,7 +483,16 @@ class SmartSearchBar extends React.Component {
       });
 
       try {
-        const values = await this.props.onGetTagValues(tag, query);
+        const {location} = this.context.router;
+        const endpointParams = {};
+
+        if (location && location.query){
+          endpointParams.statsPeriod = location.query.statsPeriod ? location.query.statsPeriod : "14d";
```
Comment (dashed) [inline src/sentry/static/sentry/app/components/smartSearchBar/index.jsx]: @wmak we have a really nice function to derive `statsPeriod`, `start`, and `end` values: https://github.com/getsentry/sentry/blob/16452ca994e81e33e108c4dafa356b6bc01d0994/src/sentry/static/sentry/app/components/organizations/globalSelectionHeader/getParams.tsx#L109-L142

usage is something like:

```js
const {start, end, statsPeriod} = getParams(location.query);
```

**Review by dashed — CHANGES_REQUESTED** [review]

File: src/sentry/static/sentry/app/components/smartSearchBar/index.jsx (hunk 1 above)
Comment (wmak) [reply src/sentry/static/sentry/app/components/smartSearchBar/index.jsx]: 🙏 added!

File: src/sentry/static/sentry/app/components/smartSearchBar/index.jsx [hunk 2]
```diff
@@ -494,6 +504,7 @@ class SmartSearchBar extends React.Component {
           };
         });
       } catch (err) {
+        console.log(err);
```
Comment (dashed) [inline src/sentry/static/sentry/app/components/smartSearchBar/index.jsx]: @wmak stray console.log

**Comment by fpacifici** [comment]
I think the python broken tests are related

**Review by dashed — APPROVED** [review]
Looks good to me 👍

**Pull request merged**